keras_cv/layers/preprocessing/vectorized_base_image_augmentation_layer_test.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
2023 -- also, should this be marked as branched from the old base layer test for better blame history?

Reply:
Done -- can you paste me the command to accomplish this?